android/src/main/java/com/stripeterminalreactnative/StripeTerminalReactNativeModule.kt

```diff
@@ -537,12 +541,13 @@ class StripeTerminalReactNativeModule(reactContext: ReactApplicationContext) :
             }
 
             val customerConsentCollected = getBoolean(params, "customerConsentCollected")
+            val enableCustomerCancellation = getBoolean(params, "enableCustomerCancellation")
```

Comment:
Does `getBoolean()` default to `false` if it's not provided?

Reply:
yep https://github.com/stripe/stripe-terminal-react-native/blob/abccbdf4/android/src/main/java/com/stripeterminalreactnative/Mappers.kt#L41-L42